Misc.GetPadding crashed

```
func TestMisc_GetPadding(t *testing.T) {
	Init(nil)
	m := NewImage()
	m.SetPadding(1, 2)
	x := 0
	y := 0
	m.GetPadding(&x, &y)
	assert.Equal(t, x, 1)
	assert.Equal(t, y, 2)
}
```

crash as follows
```
--- FAIL: TestMisc_GetPadding (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x42e210d]

goroutine 9 [running]:
testing.tRunner.func1(0xc4200fc4b0)
	/usr/local/Cellar/go/1.9.1/libexec/src/testing/testing.go:711 +0x2d2
panic(0x4386a60, 0x463eda0)
	/usr/local/Cellar/go/1.9.1/libexec/src/runtime/panic.go:491 +0x283
github.com/mattn/go-gtk/gtk.(*Misc).GetPadding(0xc42000e0e0, 0xc42002df78, 0xc42002df70)
	/Users/lidaobing/go/src/github.com/mattn/go-gtk/gtk/gtk.go:10239 +0x4d
github.com/mattn/go-gtk/gtk.TestMisc_GetPadding(0xc4200fc4b0)
	/Users/lidaobing/go/src/github.com/mattn/go-gtk/gtk/gtk_test.go:101 +0x90
testing.tRunner(0xc4200fc4b0, 0x4402518)
	/usr/local/Cellar/go/1.9.1/libexec/src/testing/testing.go:746 +0xd0
created by testing.(*T).Run
	/usr/local/Cellar/go/1.9.1/libexec/src/testing/testing.go:789 +0x2de
FAIL	github.com/mattn/go-gtk/gtk	0.858s
```

--- a/gtk/gtk.go
+++ b/gtk/gtk.go
@@ -10227,17 +10227,17 @@ func (m *Misc) SetPadding(xpad, ypad int) {
 }
 
 func (m *Misc) GetAlignment(xalign, yalign *float64) {
-	var gxalign, gyalign *C.gfloat
-	C.gtk_misc_get_alignment(MISC(m), gxalign, gyalign)
-	*xalign = float64(*gxalign)
-	*yalign = float64(*gyalign)
+	var gxalign, gyalign C.gfloat
+	C.gtk_misc_get_alignment(MISC(m), &gxalign, &gyalign)
+	*xalign = float64(gxalign)
+	*yalign = float64(gyalign)
 }
 
 func (m *Misc) GetPadding(xpad, ypad *int) {
-	var gxpad, gypad *C.gint
-	C.gtk_misc_get_padding(MISC(m), gxpad, gypad)
-	*xpad = int(*gxpad)
-	*ypad = int(*gypad)
+	var gxpad, gypad C.gint
+	C.gtk_misc_get_padding(MISC(m), &gxpad, &gypad)
+	*xpad = int(gxpad)
+	*ypad = int(gypad)
 }
 
 //-----------------------------------------------------------------------

--- a/gtk/gtk_test.go
+++ b/gtk/gtk_test.go
@@ -91,3 +91,14 @@ func TestEntry_SetInnerBorder(t *testing.T) {
 	e.SetInnerBorder(nil)
 	assert.Nil(t, e.GetInnerBorder())
 }
+
+func TestMisc_GetPadding(t *testing.T) {
+	Init(nil)
+	m := NewImage()
+	m.SetPadding(1, 2)
+	x := 0
+	y := 0
+	m.GetPadding(&x, &y)
+	assert.Equal(t, x, 1)
+	assert.Equal(t, y, 2)
+}

